<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDM</class>
 <widget class="QMainWindow" name="PDM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vital Health</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="trainButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Training Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="liveButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Live Mode</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
